joint_state_controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 20

diff_drive_controller:
  type: "diff_drive_controller/DiffDriveController"
  left_wheel: 'wheel_left_joint'
  right_wheel: 'wheel_right_joint'
  pose_covariance_diagonal:  [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 0.03]
  twist_covariance_diagonal: [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 0.03]
  wheel_separation: 0.175
  wheel_radius: 0.035
  base_frame_id: 'base_footprint'
